src/screens/UserPortal/Chat/Chat.tsx

```diff
@@ -105,29 +113,12 @@ export default function chat(): JSX.Element {
     },
   });
 
-  React.useEffect(() => {
+  useEffect(() => {
     if (chatsListData) {
       setChats(chatsListData.chatsByUserId);
```

Comment:
Why was this file edited for functionality outside of the CSS? We have an open PR to fix chat.